src/main/java/com/immunization/reference/model/PatientDetails.java

```diff
@@ -0,0 +1,249 @@
+package com.immunization.reference.model;
+
+public class PatientDetails {
+
+  private String mrn;
+  private String mrnAuthority;
+  private String nameFirst;
+  private String nameMiddle;
+  private String nameLast;
+  private String nameType;
+  private String motherMaidenNameLast;
+  private String dateOfBirth;
+  private String sex;
+  private String race;
+  private String ethnicity;
```

Comment:
We need these to be split up into raceCode/raceDescription and ethnicityCode/ethnicityDescription. 4 fields instead of 2.

Reply:
OK, I can swing that.

Reply:
done